src/backend/joanie/core/models/products.py

```diff
@@ -614,15 +614,14 @@ def target_course_runs(self):
         course_relations_with_course_runs = self.course_relations.filter(
             course_runs__isnull=False
         ).only("pk")
+        target_courses_rest = self.target_courses.exclude(
```

Comment:
what means `rest` ? I don't understand the purpose of this variable reading it.

Reply:
I mean all product's target courses except those ones implied in the course relations retrieved just above.